jetbrains-rider/src/software/aws/toolkits/jetbrains/services/lambda/dotnet/DotNetSamProjectTemplate.kt

```diff
@@ -14,7 +14,7 @@ class DotNetSamProjectTemplate : SamProjectTemplate() {
 
     override fun getDescription(): String? = message("sam.init.template.hello_world.description")
 
-    override fun supportedRuntimes(): Set<Runtime> = setOf(Runtime.DOTNETCORE2_0, Runtime.DOTNETCORE2_1)
+    override fun supportedRuntimes(): Set<Runtime> = setOf(Runtime.DOTNETCORE2_1, Runtime.DOTNETCORE3_1)
```

Comment:
Is removing 2.0 changelog-noteworthy

Reply:
I debated, ill add one. I started one and was like..hmmm and did cntrl+c